tests/e2e/nvidia-container-toolkit_test.go

```diff
@@ -74,6 +74,12 @@ var _ = Describe("docker", Ordered, func() {
 			Expect(containerOutput).To(Equal(hostOutput))
 		})
 
+		It("should support automatic CDI spec generation with the --gpus flag", func(ctx context.Context) {
```

Comment:
Judging by the CI check names I see have run on this PR -- this test wasn't executed on those, right? (I was looking for some kind of `e2e` name).

If this test ran on this PR -- can you point me to a log?

If this test did not run on this PR -- can you briefly explain where/when we run this test? Also, did you run this manually?

Reply:
I don't think that this triggered the e2e tests, and if it did, I think it probably failed. We definitely need to improve the visibility / traceability of the tests.

One issue is that we trigger the tests once the images are built, but this does not seem to trigger a "checks" entry at a PR level.